docs/source/gcm/user_guide/introduction.rst

```diff
@@ -120,8 +115,25 @@ some synthetic data instead. The API takes data in form of Pandas DataFrames:
 Note how the columns X, Y, Z correspond to our nodes X, Y, Z in the graph constructed above. We can also see how the
 values of X influence the values of Y and how the values of Y influence the values of Z in that data set.
 
-In the real world, this data comes as an opaque stream of values, where we don't know how one
-variable influences another. The SCM-based  can basically help us to deconstruct these causal
+The causal model created above allows us now to assign causal mechanisms to each node in the form of functional causal
+models. Here, these mechanism can either be assigned manually if, for instance, prior knowledge about certain causal
+relationships are known or they can be assigned automatically using the :mod:`~dowhy.gcm.auto` module. For the latter,
+we simply call:
+
+>>> gcm.auto.assign_causal_mechanisms(causal_model, data)
+
+In case we want to have more control over the assigned mechanisms, we can do this manually as well. For instance, we can
+can assign an empirical distribution model to the root node X and linear additive noise models to nodes Y and Z:
```

Comment:
We could remove "model" from "empirical distribution model.

Reply:
I updated the text and removed 'model'.